tests/Google/Ads/GoogleAds/InstantiateClassesTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * Copyright 2019, Google Inc.
```

Comment:
I believe this should be LLC already.
Could you copy from our latest file like in examples/?

Reply:
Ow, you're right, I copied the whole copyright block from an example, should be ok now.

Thanks for spotting.